shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/DockerStorageContainer.java

```diff
@@ -50,14 +50,17 @@ public abstract class DockerStorageContainer extends DockerITContainer implement
     @Getter(AccessLevel.NONE)
     private final String scenario;
     
+    private final String module;
+    
     private final Map<String, DataSource> actualDataSourceMap;
     
     private final Map<String, DataSource> expectedDataSourceMap;
     
-    public DockerStorageContainer(final DatabaseType databaseType, final String dockerImageName, final String scenario) {
+    public DockerStorageContainer(final DatabaseType databaseType, final String dockerImageName, final String scenario, final String module) {
```

Comment:
The module is not make sense, the basic module should not know other module name.
How about merge module and scenario?

Reply:
this is not a replacement, it's polymorphism. what I thought is refactor scaling and then refactor transaction, finally refactor suite. if we merge module and scenario now, this will affect too much.

Reply:
and for the combination for modue and scenario. I prefer to design a InitStrategy. likeMySQL ScalingStrategy, the content of the strategy in this module will implement by scaling developer, they will decide how to init storage type cotnainer , if this is no special init strategy, we will provide a MySQLDefaultStrategy which just init a common my.cnf and related init sql.

Reply:
> The module is not make sense, the basic module should not know other module name. How about merge module and scenario?

after all container management and config reclaimed in env, this may refactored as a`StorageContainerContext` .

Reply:
done